views/js/qtiCreator/widgets/helpers/validators.js

```diff
@@ -73,7 +73,7 @@ define([
         //warning: simplistic implementation, allow only one unique identifier in the item no matter the element class/type
         {
             name: 'availableIdentifier',
-            message: __('This identifier must not be used by any other response or item variable.'),
```

Comment:
Why do not delete the translations for the old message? Is it still used anywhere?

Reply:
it will be deleted after sync with Crowdin